conductor_api/src/conductor/base.rs

```diff
@@ -607,6 +608,34 @@ impl Conductor {
                     ))
                 })?;
 
+                // This is where we are checking the consistency between DNA: for now we compare
+                // the hash provided in the TOML Conductor config file with the computed hash of
+                // the loaded dna.
+                {
+                    // let dna_conf_hash = dna_config.hash.clone().unwrap_or("".to_string());
+                    let dna_hash_from_conductor_config =
+                        HashString::from(dna_config.hash.clone().unwrap_or("".to_string()))
+                            .to_owned();
+                    // dbg!(dna_conf_hash);
+                    let dna_hash_computed = &dna.address();
+                    // dbg!(loaded_dna_hash);
+                    Conductor::check_dna_consistency(
+                        &dna_hash_from_conductor_config,
+                        &dna_hash_computed,
+                    )
+                    // .map_err(|_| {
+                    .unwrap_or({
```

Comment:
Looks like the error message from the `check_dna_consistency` function is being thrown away. Do I have that right?

Reply:
That was just a WIP, the actual implementation was still undefined at that time ^^'